include/rocksdb/table.h

```diff
@@ -285,6 +285,25 @@ struct BlockBasedTableOptions {
   // separately
   uint64_t metadata_block_size = 4096;
 
+  // If true, an estimated memory usage of table building
```

Comment:
"an estimated memory usage" - As stated, it's not totally clear that we have a dynamically updating charge loosely based on actual usage.